presto-tests/src/test/java/io/prestosql/tests/TestDistributedEngineOnlyQueries.java

```diff
@@ -109,24 +110,23 @@ public void testRowTypeWithReservedKeyword()
     @Test
     public void testExplainAnalyze()
     {
-        assertExplainAnalyze(
+        assertThat((String) computeScalar(
```

Comment:
See my comment: https://github.com/prestosql/presto/pull/6139#issuecomment-735453667. Explain analyze assertion can be generalized.

Why wouldn't we care about matching of basic `(?s:.*)CPU:.*, Input:.*, Output(?s:.*)` for these explains and about improving matching further (TODO in original `assertExplainAnalyze` method)?

Reply:
> See my comment: #6139 (comment). Explain analyze assertion can (and should) be generalized.

I totally agree we should improve EXPLAIN ANALYZE test coverage. Whether in generalized manner, or targeted test cases is secondary concern to me.